rust/opendp/src/meas/gaussian.rs

```diff
@@ -31,6 +31,7 @@ fn make_gaussian_privacy_relation<T: 'static + Clone + SampleGaussian + Float>(s
 
 
 pub trait GaussianDomain: Domain {
+    type Metric: SensitivityMetric<Distance=Self::Atom> + Default;
```

Comment:
If default associated types were stable, you could do this:
```rust
type Metric: SensitivityMetric + Default;
type Atom = Self::Metric::Distance;
```
Then you could avoid setting Atom from the impls, and Atom becomes essentially a type alias for Self::Metric::Distance. You can still drop the Atom, but it would make references to Atom really wordy.